### PR TITLE
Add vmod_str in 6.0-lts branch

### LINKED ISSUE #175: Provide vmod_str in 6.0-lts branch ?
The (now frozen) https://github.com/gquintard/libvmod-str repo provides branches for 4.0 / 4.1 / 5.0 / 5.1, and then before a 6.0 branch was published the code was moved here in varnish-modules but only for 6.5+

Would it be possible to provide a version of vmod_str for the 6.0 LTS ?

### PR DESCRIPTION
**Pull request by yched**
Fixes #175

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-dist: trusty
+dist: bionic
 
 language: c
 
 before_install:
   - sudo apt-get update -q
   - sudo apt-get install -qq apt-transport-https python-docutils
-  - curl -s https://packagecloud.io/install/repositories/varnishcache/varnish41/script.deb.sh | sudo bash
+  - curl -s https://packagecloud.io/install/repositories/varnishcache/varnish60/script.deb.sh | sudo bash
```
Comment (gquintard) [inline .travis.yml:10]: we actually use https://packagecloud.io/varnishcache/varnish60lts for this one

**Review by gquintard — CHANGES_REQUESTED** [review]
thank you for the quick PR, I just have one note about the packagecloud repo, but once this is fixed, you can squash, force-push and I'll merge.

File: .travis.yml (hunk 1 above)
Comment (yched) [reply .travis.yml]: ah right - done :)

**Comment by gquintard** [comment]
in it goes, thanks!

**Comment by yched** [comment]
w00t, that was fast :-D thanks !

6.0-lts was released as 0.15.1 so far, does that mean there will be a 0.15.2 with vmod_str ?
(although I see that master's vmod_str for 6.5 is not in a tagged released yet either, so yeah, not sure how you want to plan this)

**Comment by gquintard** [comment]
you did the work, and there's was really no reason to linger.

As for releases, we usually tag them once Varnish is release, so March 15th and September 15th., unless there's a big, pressing issue

**Pull request merged**